src/normalizer.ts

```diff
@@ -43,6 +43,30 @@ rules.set('Default top level `id`', (schema, rootSchema, fileName) => {
   return schema
 })
 
+function escape(
+  code: string | { [key: string]: any },
+  replacer = '*\\/'
+) {
+  if (code === null || typeof code !== 'object') {
+    return
+  }
+  for (const key of Object.keys(code)) {
+    if (key === 'description' && typeof code[key] === 'string') {
+      code[key] = code[key].replace(/\*\//g, replacer)
+    } else {
+      escape(code[key])
```

Comment:
I don't think you need to manually recurse. IIRC `mapDeep` handles this for you.